WTWSMS/decisions/VIET_Eastern_religions_decisions.txt

```diff
@@ -31,7 +32,7 @@ decisions = {
 					trait = kind
 				}
 			}
-			NOR = {
+			NOT = {
```

Comment:
@loup99 

It was an error. "NOT" tells the code that if any one of those conditions are met, the action won't be applied or enabled. "NOR" is not a command. It was a typo.